backend/src/main/java/de/neuefische/capstone/backend/models/Category.java

```diff
@@ -0,0 +1,6 @@
+package de.neuefische.capstone.backend.models;
+
+public enum Category {
```

Comment:
`
public enum Category{
Donation("donation"), 
PARTICIPATION("participation");
private final String cat;
Category(String c){
cat = c
}
public String getCategory()
{
return cat;
}
}
`
Du hast die enum gut geschrieben aber nur ein Tip um  Problem mit dem Type in frontend zu vermeiden willst du dein enum als String und nicht als enum Type.

in deine Model kannst du jetzt so schreiben
`String Category;`

und in Service wird dann so verwendet z.B um die Category als String zu kriegen
`Category.DONATION.getCategory()`
oder
`Category.PARTICIPATION.getCategory()`

Reply:
Danke für den Tipp Collins! 👍🏻